packages/grafana-data/src/transformations/transformers/fieldConversion.test.ts

```diff
@@ -0,0 +1,152 @@
+import { toDataFrame } from '../../dataframe/processDataFrame';
+import { FieldType } from '../../types/dataFrame';
+import { mockTransformationsRegistry } from '../../utils/tests/mockTransformationsRegistry';
+import { ensureTimeField, fieldConversion, fieldConversionTransformer } from './fieldConversion';
+
+const stringTime = toDataFrame({
```

Comment:
for the tests, it is nicer to see the input and output closer... maybe rather than defined constants at the top, just stick them inline?